search/search-domain/src/main/java/io/camunda/search/entities/TenantMemberEntity.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.search.entities;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public record TenantMemberEntity(String id, TenantMemberType tenantMemberType) {
+  public enum TenantMemberType {
```

Comment:
Is this the same as `io.camunda.zeebe.protocol.record.value.EntityType`?

Reply:
Yea, I couldn't find a nice place to lift `EntityType` and wanted to keep the PR smaller (without introducing a shared module other than the shared security module etc). 

I can look at this but at the moment it seems like there isn't a nice (non security scoped) method of sharing these common enums between both the engine and the search classes

Reply:
Ah, I've just seen that the search query transformers module has a dependency to the zeebe protocol module so I'm just going to add that in here for the search domain module